src/CoreWCF.Primitives/src/CoreWCF/Security/IListenerSecureConversationSessionSettings.cs

```diff
@@ -0,0 +1,44 @@
+
+using System;
+
+namespace CoreWCF.Security
+{
+    interface IListenerSecureConversationSessionSettings
```

Comment:
>IListenerSecureConversationSessionSettings [](start = 14, length = 42)

Rename to IServiceDispatcherSecureConversationSerssionSettings. Also add the internal modified